src/adc.py

```diff
@@ -55,7 +55,7 @@ def read_leak(self):
 
     def read_pH(self):
         pH_voltage = self.pH_sensor.voltage
-        pH = 7.7 +(pH_voltage-1.65)*(-3.3)
+        pH = 7.7 +(pH_voltage-0.82)*(-3.3) #formula adjusted for use with a voltage divider to map the 5 V output to 3.3V for use with a 3.3V ADC
```

Comment:
It would be good to also include these as contants. That way they are more readable in code, and easier to change. The 3.3 and 7.7 here. Maybe we also want to include a `calibrate_PH` function if we do that from the app later?

Reply:
oooh a calibrate pH function is a good idea. Will think more on this.